pkg/dashboard/static/repo.js

```diff
@@ -130,6 +130,11 @@ function repoChartClicked() {
         setHashParam("chart", elm.installed_name)
         window.location.reload()
     } else {
-        popUpUpgrade(elm)
+        $.getJSON("/api/kube/contexts").fail(function (xhr) {
```

Comment:
Instead of doing another API call, let's save that info into a data attribute and just use here. We already do query contests list in the page load.